src/features/donations/components/Donation/ShareDonation/ShareButton.module.scss

```diff
@@ -0,0 +1,88 @@
+.share-button {
+  border: 1px solid #fff;
+  border-radius: 5px;
+}
+
+.share-button button {
+  width: 57px;
+  height: 47px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.share-button__whatsapp {
+  &:hover {
+    svg path {
+      fill: #25d366;

```

Comment:
what do you think about adding these colors to the constants file?

Reply:
There are many colors (are diferent for hover or active). If you think that is the best, I apply that.

Reply:
suggestions for colors names? are diferents for active or hover

Reply:
Not sure but you could use the name of the social media, when is active you could add a --active 👇🏻 

```scss
$faceboook: #fff;
$facebook--active: #000;
```